app/views/document-response.njk

```diff
@@ -52,7 +52,9 @@
           </button>
         </div>
       </div>
-      <p id="Correspondence" class="govuk-body" style="display: none;">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu</p>
+      <p id="Correspondence" class="govuk-body" style="display: none;">
+        {{ contents | replace("\n", "<br>") | safe }}
```

Comment:
So just a general point here: We probably shouldn't be piping into the `safe` filter as we can't 100% trust the contents.

Not something to fix here as it's a wider issue across the repo which will need addressed in another ticket. We probably should split the content on the backend by `\n` and use the `join` filter and set the separator to `<br>`.

Happy to discuss.